core/src/avm2/globals/xml.rs

```diff
@@ -477,7 +474,7 @@ pub fn text<'gc>(
     let nodes = if let E4XNodeKind::Element { children, .. } = &*xml.node().kind() {
         children
             .iter()
-            .filter(|node| matches!(&*node.kind(), E4XNodeKind::Text(_)))
+            .filter(|node| node.is_text())
```

Comment:
This now matches CData as well as text - is that intentional?

Reply:
Cdata is text. The previous code was wrong.